reference/fleet/tls-overview.md

```diff
@@ -97,3 +103,35 @@ Note that you can also configure mutual TLS for {{fleet-server}} and {{agent}} [
 :alt: Diagram of mutual TLS connection between components
 :::
 
+## Configure TLS/mTLS settings in the Fleet UI [tls-ui-settings]
+```{applies_to}
+  stack: ga 9.1
+```
+
+You can configure TLS and mutual TLS (mTLS) settings for {{fleet-server}} hosts using the {{fleet}} UI instead of CLI flags. This approach simplifies certificate configuration.
+
+To access these settings:
+
+1. In **Kibana**, go to **Management > {{fleet}} > Settings**.
+2. Under **Fleet Server hosts**, select **Add host** or edit an existing host.
+3. Expand the **SSL options** section.
+
+### SSL options
```

Comment:
I see where the mix up is - the` elastic-agent-cert` and  `--elastic-agent-cert-key` are the ones available under `Fleet settings > output` . We should point the user to that section if they want to set up a mTLS connection